src/plugins/data_source_management/public/components/data_source_table/data_source_table.tsx

```diff
@@ -4,15 +4,30 @@
  */
 
 import { EuiTitle } from '@elastic/eui';
-import React from 'react';
-import { withRouter } from 'react-router-dom';
+import React, { useEffect } from 'react';
+import { i18n } from '@osd/i18n';
+import { ChromeBreadcrumb } from 'src/core/public';
 
-export const DataSourceTable = () => {
+export interface DataSourceTableProps {
+  setBreadcrumbs: (crumbs: ChromeBreadcrumb[]) => void;
+}
+
+export const DataSourceTable = ({setBreadcrumbs}: DataSourceTableProps) => {
+  useEffect(() => {
+    setBreadcrumbs([
```

Comment:
how about we put all the breadcrumbs in one single file and import each individual one on it's corresponding page

Reply:
done